extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -53,6 +55,27 @@ void build(CombinedIndexBuildItem index,
             BuildProducer<GeneratedBeanBuildItem> generatedBeans,
             BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
 
+        Config config = ConfigProvider.getConfig();
+        Optional<String> springJpaConfig = config.getOptionalValue("spring.jpa.show-sql", String.class);
+        Map<String, String> springJpaPropertiesMap = new HashMap<>();
```

Comment:
Maybe a more prescriptive name would help?

Reply:
Do you mean the name of the `springJpaPropertiesMap`? 
WDYT about `springJpaToQuarkusOrmPropertiesMap`?

Reply:
Yeah, that's what I had in mind :)

Reply:
thank you @geoand